R/client-connect.R

```diff
@@ -61,18 +68,20 @@ connectClient <- function(service, authInfo) {
         details$title <- title
       }
 
-      # RSC doesn't currently use the template or account ID
+      # Connect doesn't use the template or account ID
       # parameters; they exist for compatibility with lucid.
-      application <- POST_JSON(service, authInfo, "/applications", details)
+      result <- POST_JSON(service, authInfo, v1_url("content"), details)
       list(
-        id = application$id,
-        guid = application$guid,
-        url = application$url
+        id = result$id,
+        guid = result$guid,
+        url = result$content_url,
+        # Include dashboard_url so we can open it or logs path after deploy
+        dashboard_url = result$dashboard_url
       )
     },
 
-    uploadApplication = function(appId, bundlePath) {
-      path <- file.path("/applications", appId, "upload")
+    uploadBundle = function(contentGuid, bundlePath) {
```

Comment:
Function is renamed and swapped to taking guid. Other similar functions for shinyapps and Connect Cloud already refer to bundles.